README.md

```diff
@@ -140,6 +139,15 @@ sudo apt-get install -y \
         file \
         zlib1g-dev
 ```
+
+On Ubuntu 20.04, update to gcc-10 which has C++20 support:
+```bash
+sudo apt-get install -y gcc-10 g++-10 cpp-10
+sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 100 --slave \
```

Comment:
I don't think we should make system-wide global alternations to a user's environment like this. Can we just document it like we did in the past for Ubuntu 18? (`CC=gcc-10 CXX=g++-10 cmake ...`)

Reply:
done